pkg/monitor/worker.go

```diff
@@ -104,6 +104,8 @@ func (mon *monitor) changefeed(ctx context.Context, baseLog *logrus.Entry, stop
 			mon.mu.Unlock()
 		}
 
+		mon.lastChangefeed = time.Now()
```

Comment:
this will fire if the Next() calls were unsuccessful.  I don't think it should.

Reply:
added check for errors on both .Next calls.